src/App.js

```diff
@@ -94,6 +98,7 @@ class App extends Component {
       <Router history={browserHistory}>
         <Route path="/button" component={Home}/>
         <Route path="/" component={ActivityPage}/>
+        <Route path="/Title" component={TitlePage}/>
```

Comment:
should be lowercase t but I think it should also just be the home page, when @morgancbest adds her page navigation in then those can be linked from the title page